src/app/features/profile/pages/edit-profile/edit-profile.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+import { IUser } from '@core/models/User';
+import { passwordValidation } from '@shared/validators/validations';
+
+@Component({
+	selector: 'app-edit-profile',
+	templateUrl: './edit-profile.component.html',
+	styleUrls: ['./edit-profile.component.scss']
+})
+export class EditProfileComponent implements OnInit {
+	changeProfileForm: any;
```

Comment:
Try to avoid : any type, as much ass possible.

Reply:
Fixed it, ty for review